tests/nuodb_test_class.py

```diff
@@ -34,15 +35,36 @@ def assert_await(fxn, timeout=120, interval=10):
 
     if last_assert:
         raise last_assert
+    
+def getClient():
+     influxToken = os.getenv("INFLUXDB_TOKEN", 'none')
```

Comment:
The test should fail fast if this is not set as there is no way to authenticate.

Reply:
Should I provide token as the default value ?

Reply:
If `INFLUXDB_TOKEN` environment variable is not found, we should just fail the test.

Reply:
The test will fail automatically with the unauthenticated error message.